app/observatory/burst/healthping_result.go

```diff
@@ -87,13 +89,20 @@ func (h *HealthPingRTTS) getStatistics() *HealthPingStats {
 	cnt := 0
 	validRTTs := make([]time.Duration, 0)
 	for _, rtt := range h.rtts {
+		timestamp := rtt.time.Unix()
+		if timestamp > stats.LastTryTime {
+			stats.LastTryTime = timestamp
```

Comment:
Thanks for your work. Although, I think lastTryTime and LastSeenTime should be the time when we probe, not when we check the result.